packages/gatsby-cli/src/create-cli.js

```diff
@@ -82,7 +82,7 @@ function buildLocalCommands(cli, isLocalSite) {
   function getCommandHandler(command, handler) {
     return argv => {
       report.setVerbose(!!argv.verbose)
-      if (argv.noColor) {
+      if (argv.noColor || process.env.NO_COLOR) {
```

Comment:
On line 92 there is another `argv.noColor` usage. So I would put the noColor in a variable so we can re-use it in the function.
```suggestion
      const noColor = argv.noColor || process.env.NO_COLOR
      if (noColor ) {
```

Reply:
Thanks for your feedback, I refactored the section in https://github.com/gatsbyjs/gatsby/pull/16372/commits/ad4c029f6a2f131f97cdee319d792fd3c12713d1.

Another suggestion from my side is to put [lines 87-90](https://github.com/gatsbyjs/gatsby/pull/16372/commits/ad4c029f6a2f131f97cdee319d792fd3c12713d1#diff-dfccf547543bb987b0ff0a7281348933R87-R90) into the `reporter.setNoColor()` function, since it is just another internal color setting. What do you think?

Reply:
oh yeah that sounds great! can you do that refactor in this pr? 😄